### PR TITLE
Fix tribe image resource URLs in app shop | spotfix

### PR DESCRIPTION
**Pull request by barryhughes**
Fix broken app shop images (pass the Tribe__Main object so it can figure out the paths from there).

### CODE REVIEW

File: src/admin-views/app-shop.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 	<div class="header">
 		<h1><?php esc_html_e( 'Tribe Event Add-Ons', 'tribe-common' ); ?></h1>
-		<a href="https://theeventscalendar.com/?utm_campaign=in-app&utm_source=addonspage&utm_medium=top-banner" target="_blank"><img src="<?php echo esc_url( tribe_resource_url( 'images/app-shop-banner.jpg', false, 'common' ) ); ?>" /></a>
+		<a href="https://theeventscalendar.com/?utm_campaign=in-app&utm_source=addonspage&utm_medium=top-banner" target="_blank"><img src="<?php echo esc_url( tribe_resource_url( '/images/app-shop-banner.jpg', false, null, $main ) ); ?>" /></a>
```
Comment (bordoni) [inline src/admin-views/app-shop.php]: Shouldn't we use the `Tribe__Main::$tec_url` here to compose this URL? Or maybe a `m.tri.be` link

Comment (barryhughes) [inline src/admin-views/app-shop.php]: We probably should - I think I left the same feedback in the PR that added those links - but in this case all I'm doing is correcting the resource URL for the images.

Comment (bordoni) [inline src/admin-views/app-shop.php]: As we talked I will ping Zach about these, on the On going discussion.

**Comment by bordoni** [comment]
👍

**Pull request merged**